### PR TITLE
Clear GSheets before updating values

### LINKED ISSUE #45: Remove GSheets content before writing
**Describe the bug**
When writing into a GSheets document, we should check if there are some values already written and remove the content in this case. If we don't we could end up with some previous content left.
One example for this is writing the dao round winners.

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #45 .

### CODE REVIEW

**Comment by KatunaNorbert** [comment]
For proper tasting you can set a breakpoint before the `emptySheet()` method inside `updateValues()` method and configure Airtable in such way that running `update_funding_round` script, you will be on the 'Start next round' case. And then run the script with the debugger so you can allow update only after you check that the sheet is cleared.

File: src/gsheets/gsheets_utils.js [hunk 1]
```diff
@@ -47,9 +47,24 @@ const addSheet = async (oAuth, sheetName, indexOffset=0) => {
...
+    } catch(err) {
+        console.log('[getValues] The API returned an error: ' + err);
+        return undefined
+    }
+}
+
 // Dump values to Sheet
 const updateValues = async (oAuth, sheetId, range, values) => {
     const sheets = google.sheets({version: 'v4', auth: oAuth});
+    await emptySheet(oAuth, sheetId, range)
```
Comment (idiom-bytes) [inline src/gsheets/gsheets_utils.js:67]: I believe this is what I was looking for.
Clear(range) then DumpTo(range)

Comment (idiom-bytes) [reply src/gsheets/gsheets_utils.js:67]: I had noticed that previous commits were calling:
1. emptySheet()
2. updateValues()

Thanks for implementing this at the main entry point. 
Seems like a great place for it! Awesome

**Review by idiom-bytes — APPROVED** [review]
Thank you Norbert. 

I noticed you implemented it in a different way previously, but then refactored! I believe calling [emptySheet()], inside of the common function [updateValues()] is much better too!

Awesome!

**Comment by idiom-bytes** [comment]
> For proper tasting you can set a breakpoint before the emptySheet() method inside updateValues() method and configure Airtable in such way that running update_funding_round script, you will be on the 'Start next round' case. And then run the script with the debugger so you can allow update only after you check that the sheet is cleared.

@KatunaNorbert @claudiaHash how obvious is it to run tests this way? Was this easy to grok? I'm wondering how we can improve our documentation on some of these things are done.

Finally.... Looks great Norbert! Thank you!

File: src/gsheets/gsheets_utils.js (hunk 1 above)
Comment (KatunaNorbert) [reply src/gsheets/gsheets_utils.js:67]: Yep, was taking the issue step by step testing in different places and only in the end when I saw the bigger picture I realized that we could simplify it

**Comment by KatunaNorbert** [comment]
Well not the easiest way to setup a test, but is quite hard to test the clear and update methods otherwise because it happens very fast. I could set up a test for clearing a sheet but If I combine it with update it would happen to fast and don't get the chance to se the sheet cleared

**Review by AnaLoznianu — APPROVED** [review]
Looks good to me

**Pull request merged**